plugins/out_s3/s3.c

```diff
@@ -119,6 +119,40 @@ static char *mock_error_response(char *error_env_var)
     return NULL;
 }
 
+static void s3_retry_warn(struct flb_s3 *ctx, char* tag, 
+                          char* input_name, time_t create_time, 
+                          int less_than_limit)
+{
+    struct tm now_time;
+    char create_time_str[20];
+    struct tm *tmp;
+    tmp = localtime_r(&create_time, &now_time);
+    strftime(create_time_str, 20, "%Y-%m-%d %H:%M:%S", tmp);
+    if (input_name == NULL) {
+        if (less_than_limit == FLB_TRUE) {
+            flb_plg_warn(ctx->ins,
+                        "failed to flush chunk tag=%s, create_time=%s"
+                        "(out_id=%d)",
+                        tag, create_time_str, ctx->ins->id);
+        } else {
```

Comment:
`else` must be in a new line` (applies for other lines too)

Reply:
will follow this code style this time and later. Thank you for your detailed review!